app-editors/gvim/gvim-9.0.0099.ebuild

```diff
@@ -0,0 +1,352 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-editors/vim-core and app-editors/vim
+
+VIM_VERSION="9.0"
+LUA_COMPAT=( lua5-{1,3,4} luajit )
```

Comment:
You can use ```1..4```, as lua5_2 is anyway globally masked.

Reply:
done.